package-lock.json

```diff
@@ -487,12 +487,12 @@
     },
     "array-includes": {
       "version": "3.0.3",
-      "resolved": "https://registry.npmjs.org/array-includes/-/array-includes-3.0.3.tgz",
+      "resolved": "https://artifactory.cloud.capitalone.com:443/artifactory/api/npm/npm-internalfacing/array-includes/-/array-includes-3.0.3.tgz",
```

Comment:
I assume we don't need any of the changes in this file? 😉I didn't realized you work for a bank!

Reply:
Whoops, I'll change it back. Yeah, we're hoping to use this here. I'll revert the entire `package-lock.json`.